modules/extensions/galasa-extensions-parent/dev.galasa.extensions.common.couchdb/src/main/java/dev/galasa/extensions/common/couchdb/CouchdbStore.java

```diff
@@ -158,6 +159,9 @@ public <T> T getDocumentFromDatabase(String dbName, String documentId, Class<T>
     public ViewResponse getDocumentsFromDatabaseViewByKey(String dbName, String viewName, String queryKey, boolean includeDocs)
             throws CouchdbException {
         ViewResponse viewResponse = null;
+        
+        
+
```

Comment:
Really minor - personally, I'd avoid adding this much whitespace to existing code